rebasehelper/specfile.py

```diff
@@ -331,7 +331,7 @@ def get_not_used_patches(self):
         """
         return self.patches['not_applied']
 
-    def _comment_out_patches(self, patch_num, note=None):
+    def _comment_out_patches(self, patch_num, note=None, remove_patches=False):
```

Comment:
I think this method and its arguments should be changed to something more sensible. Perhaps something like `_process_patches(self, comment_out=[], remove=[], note=None)` with lists of patches to comment-out and/or remove?